.drone.yml

```diff
@@ -316,6 +316,24 @@ matrix:
       NEED_SERVER: true
       NEED_INSTALL_APP: true
 
+    - PHP_VERSION: 5.6
+      OC_VERSION: daily-stable10-qa
+      TEST_SUITE: api-acceptance
```

Comment:
Add a ``cli-acceptance`` section and use it - that will be clearer when peole read it.
(probably the ``make test-acceptance-api`` does work, "by luck")